webrtc.html

```diff
@@ -1035,18 +1132,47 @@
           <dd>
             <p>The <dfn id=
             "dom-peerconnection-remotedescription"><code>remoteDescription</code></dfn>
-            attribute MUST return the last <code><a>RTCSessionDescription</a></code>
-            that was successfully set using <code><a href=
-            "#dom-peerconnection-setremotedescription">setRemoteDescription()</a></code>,
-            plus any remote candidates that have been supplied via
+            attribute  MUST return <code><a href=
+            "#dom-peerconnection-pendingRemoteDesc">pendingRemoteDesc</a></code>
+            if it is not null and otherwise it MUST return <code><a href=
+            "#dom-peerconnection-currentremotedesc">currentRemoteDesc</a></code>. </p>
+          </dd>
+
+          <dt>readonly attribute RTCSessionDescription? currentRemoteDesc</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-peerconnection-currentremotedesc"><code>currentRemoteDesc</code></dfn>
+            attribute MUST return the last remote
+            <code><a>RTCSessionDescription</a></code> that was successfully
+            negotiated the last time the PeerConnection transitioned into the
+            stable state plus any local candidates that have been upplied via
             <code><a href=
             "#dom-peerconnection-addicecandidate">addIceCandidate()</a></code>
-            since then.</p>
+            since then. This is set by <code><a href=
+            "#dom-peerconnection-setremotedescription">setRemoteDescription()</a></code>.
+            </p>
 
             <p>A null object will be returned if the remote description has not
             yet been set.</p>
           </dd>
 
+          
+          <dt>readonly attribute RTCSessionDescription? pendingRemoteDesc</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-peerconnection-pendingRemoteDesc"><code>pendingRemoteDesc</code></dfn>
+            attribute MUST return null if the PeerConnection is in the stable
+            state and otherwise must return the remote
+            <code><a>RTCSessionDescription</a></code> that is in the process of
+            being negotiated plus any remote candidates that have been supplied
+            via <code><a href=
+            "#dom-peerconnection-addicecandidate">addIceCandidate()</a></code>
+            since then.</p>
```

Comment:
Referent for "then"?
Assuming that this is "since the setLocalDescription call was started"?